src/saveToCase.js

```diff
@@ -0,0 +1,475 @@
+                    lstring: "Properties To Create",
+                    visibility: 'visible',
+                    presence: 'optional',
+                    widget: saveCasePropWidget,
+                    validationFunc: function (mug) {
+                        if (mug.p.use_create) {
+                            var props = _.without(_.keys(mug.p.create_property), ""),
+                                required = ["case_type", "case_name"],
+                                optional = ["owner_id"],
+                                legal = _.union(required, optional),
+                                illegalProps = _.difference(props, legal),
+                                requiredProps = _.intersection(props, required),
+                                invalidProps = _.filter(props, function(p) {
+                                    return !/[a-z_]*/.test(p);
+                                });
+
+                            if (requiredProps.length !== required.length) {
+                                return "You must include " + 
+                                    required.join(", ") + 
+                                    " columns to create a case";
+                            } else if (illegalProps.length > 0) {
+                                return "You can only use the following properties: " +
+                                    legal.join(', ');
+                            } else if (invalidProps.length > 0) {
+                                return invalidProps.join(", ") + 
+                                    " are invalid properties";
+                            }
+
+                        }
+                        return 'pass';
+                    }
+                },
+                "use_close": {
+                    lstring: "Close Case",
+                    visibility: 'visible',
+                    presence: 'optional',
+                    widget: widgets.checkbox
+                },
+                "close_condition": {
+                    lstring: "Close Condition",
+                    visibility: 'visible',
+                    presence: 'optional',
+                    widget: widgets.xPath
+                },
+                "use_update": {
+                    lstring: "Update Case",
+                    visibility: 'visible',
+                    presence: 'optional',
+                    widget: widgets.checkbox
+                },
+                "update_property": {
+                    lstring: "Properties To Update",
+                    visibility: 'visible',
+                    presence: 'optional',
+                    widget: saveCasePropWidget,
+                    validationFunc: function (mug) {
+                        if (mug.p.use_update) {
+                            var props = _.without(_.keys(mug.p.update_property), ""),
+                                invalidProps = _.filter(props, function(p) {
+                                    return !/[a-z_]*/.test(p);
```

Comment:
I think this will always return false, `/[a-z_]*/` matches any string including the empty string.